op25/gr-op25_repeater/apps/tk_smartnet.py

```diff
@@ -327,11 +328,11 @@ def process_qmsg(self, msg, curr_time):
                 self.tune_next_cc()
 
         elif (m_type == 0): # OSW Receieved
-            s = msg.to_string()
+            msg_bytes = bytes(msg.to_string())
 
-            osw_addr = (ord(s[0]) << 8) + ord(s[1])
-            osw_grp  =  ord(s[2])
-            osw_cmd  = (ord(s[3]) << 8) + ord(s[4])
+            osw_addr = (msg_bytes[0] << 8) + msg_bytes[1]
+            osw_grp  =  msg_bytes[2]
+            osw_cmd  = (msg_bytes[3] << 8) + msg_bytes[4]
```

Comment:
Another option is to unpack using the struct module, without having to call `bytes()`:

`(osw_addr, osw_grp, osw_cmd) = struct.unpack('>HBH', s)`

I think this works on both py2 and py3.